scripts/website/copy_jsonschema/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	hcversion "github.com/hashicorp/go-version"
+
+	"github.com/golangci/golangci-lint/scripts/website/gh"
+)
+
+func main() {
+	dstDir := filepath.FromSlash("docs/static/jsonschema/")
+
+	err := os.RemoveAll(dstDir)
+	if err != nil {
+		log.Fatalf("remove dir: %v", err)
+	}
+
+	err = os.MkdirAll(dstDir, os.ModePerm)
+	if err != nil {
+		log.Fatalf("make dir: %v", err)
+	}
+
+	// The key is the destination file.
+	// The value is the source file.
+	files := map[string]string{}
+
+	entries, err := os.ReadDir("jsonschema")
+	if err != nil {
+		log.Fatalf("read dir: %v", err)
+	}
+
+	for _, entry := range entries {
+		if strings.HasSuffix(entry.Name(), ".jsonschema.json") {
+			files[entry.Name()] = entry.Name()
+		}
+	}
+
+	latest, err := gh.GetLatestVersion()
+	if err != nil {
+		log.Fatalf("get latest release version: %v", err)
+	}
+
+	version, err := hcversion.NewVersion(latest)
+	if err != nil {
+		log.Fatalf("parse version: %v", err)
+	}
+
+	versioned := fmt.Sprintf("golangci.v%d.%d.jsonschema.json", version.Segments()[0], version.Segments()[1])
+	files[versioned] = "golangci.jsonschema.json"
```

Comment:
will `verify` support custom gcl schema in the future?

Reply:
Maybe as a dedicated command under `custom` :thinking: 
`config` is a command related to `golangci-lint` configuration, then `verify` too.